setup.py

```diff
@@ -1,14 +1,13 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="ray_beam_runner",
+    name="ray_beam",
     packages=find_packages(where=".", include="ray_beam_runner*"),
     version="0.0.1",
     author="Ray Team",
     description="A Ray runner for Apache Beam",
```

Comment:
```suggestion
    description="An Apache Beam Runner for Ray.",
```

Reply:
Changed to `An Apache Beam Runner using Ray.`